src/platform/qt/library/LibraryModel.h

```diff
@@ -0,0 +1,78 @@
+/* Copyright (c) 2014-2017 waddlesplash
```

Comment:
Unless this has a lot in common with waddlesplash's previous version, it might make sense to remove this copyright. @waddlesplash, any opinion? Also, if you're not going to put yourself in the copyright, please put me.

Reply:
I only glanced at the code, but it seems superficially similar at best, so I don't think I have much of a copyright claim to it indeed.